### ISSUE TITLE
Missing type when adding custom model

### ISSUE THREAD

**Issue by moesphere** [ClapeyronThermo/Clapeyron.jl#154]
Hello,

when adding a custom model (PC-SAFT example from docs), an error is thrown:
![image](https://user-images.githubusercontent.com/19595387/227953126-2cedee71-092d-4a81-9625-766c8909bce4.png)

The Clapeyron version used is 0.4.6.

Code is
```julia
# Defining an abstract type for this model type
abstract type PCSAFTModel2 <: SAFTModel end

# Defining the parameters used by the model
struct PCSAFTParam2 <: EoSParam
    Mw::SingleParam{Float64}
    segment::SingleParam{Float64}
    sigma::PairParam{Float64}
    epsilon::PairParam{Float64}
    epsilon_assoc::AssocParam{Float64}
    bondvol::AssocParam{Float64}
end

# Creating a model struct called PCSAFT, which is a sub-type of PCSAFTModel, and uses parameters defined in PCSAFTParam
@newmodel PCSAFT2 PCSAFTModel2 PCSAFTParam2
```

**Comment by longemen3000** [comment]
@moesphere , can you try updating your enviroment? a new release with the proper fix to this was released (0.4.7)

**Comment by moesphere** [comment]
Yes, it is working now. Thank you @longemen3000 for fixing this in no time!
